notebooks/00_quick_start/npa_synthetic.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>same comments for imports and tests for all notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>